### PR TITLE
return dsl unserialized, re #11386

### LINKED ISSUE #11386: ES dsl query can't be serialized to json
when the `returnDsl=True` kwarg is passed, the dsl, i.e. the elasticsearch DSL-based classes that make up a query object, shouldn't be serialized to json as our json serializer doesn't currently support that. It should be sent as an unserialized instance of a query object

### PR DESCRIPTION
**Pull request by whatisgalen**
### Types of changes

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Issues Solved

Closes #11386

### Checklist

-   I targeted one of these branches:
    - [x] dev/7.6.x (under development): features, bugfixes not covered below
    - [ ] dev/7.5.x (main support): regressions, crashing bugs, security issues, major bugs in new features
    - [ ] dev/6.2.x (extended support): major security issues, data loss issues
-   [ ] I added a changelog in arches/releases
-   [ ] I submitted a PR to arches-docs (if appropriate)
-   [ ] Unit tests pass locally with my changes
-   [ ] I added tests that prove my fix is effective or that my feature works
-   [ ] My test fails on the target branch

#### Accessibility Checklist

[Developer Guide](https://arches.readthedocs.io/en/stable/developing/advanced/accessibility/)

| Topic            | Changed | Retested |
| ---------------- | ------- | -------- |
| Color contrast   |         |          |
| Form fields      |         |          |
| Headings         |         |          |
| Links            |         |          |
| Keyboard         |         |          |
| Responsive Design|         |          |
| HTML validation  |         |          |
| Screen reader    |         |          |

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

File: arches/app/views/search.py [hunk 1]
```diff
@@ -371,6 +371,8 @@ def search_results(request, returnDsl=False):
     )
 
     if response_object:
+        if returnDsl:
+            return response_object
```
Comment (apeters) [inline arches/app/views/search.py:375]:
```suggestion
        if returnDsl:
            return search_query_object.pop("query", None)
```

**Review by apeters — CHANGES_REQUESTED** [review]
I would suggest that in `base_search_view.py` that when the method `handle_search_results_query` is called that it returns a consistent object.  So, for example, when using `returnDsl=True` that the return object looks like this:
```
(None, search_query_object)
```
None being the search result, as there is not search result when requesting the dsl.

File: arches/app/views/search.py (hunk 1 above)
Comment (whatisgalen) [reply arches/app/views/search.py]: change in latest

**Review by njkim — APPROVED** [review]
Looks good, thanks.

**Pull request merged**